inc/class-markdown-hb-import.php

```diff
@@ -68,8 +139,9 @@ public static function action_wporg_cli_hb_manifest_import() {
 				$existing[] = $post;
 			}
 		}
+
 		if ( class_exists( 'WP_CLI' ) ) {
-			\WP_CLI::success( "Successfully created {$created} command pages." );
+			\WP_CLI::success( sprintf( 'Successfully created %s command pages.', $created ) );
```

Comment:
```suggestion
			\WP_CLI::success( "Successfully created {$created} command pages." );
```

Why is current implementation changed? Current code is clearer.